BCH-Wunder-Project-team-4/Wunder.io
# Event page fix

## Link to issue

#43 

## Link to feature environment:

http://localhost:3000/

## Changes proposed in this PR:

- Fixes the main issues in #43 for speakers icon and heading rendering without speaker info added.
-  If you create a new event or edit an old one, Drupal won't let you save without adding a speaker name or removing an empty Speaker Information paragraph. This prevents the speakers heading and icon rendering with zero content.
- Adjusts the map size, centering and zoom for single event page and Contact page. Map marker popup also is positioned now a bit more above the marker and has Overpass font and bigger font size.
- Change label names for Event content type and Event Speaker Information paragraph (this doesn't affect machine names) for better readability
- Update event image size to be same as parent container.

## How to test:

1. `git fetch origin eventPage-fix`
2. `git checkout eventPage-fix`
3. `lando drush cim`
4. Create event with and without speakers. Check that the rendering is okay on both cases.
5. Add offices (e.g. Helsinki and Tallinn). If you only have one office the map will be smaller (same as event page map). That's not a bug :D The bigger map will be there with 2 or more offices created in Drupal.
6. If you have events, go to respective single event page and check that the map is now smaller, centered to the marker and more zoomed in.

## Screenshots

<img width="804" alt="Screenshot 2023-12-14 at 19 06 42" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/4adc7b16-9127-42fb-95d7-497fc7d7230d">
<img width="866" alt="Screenshot 2023-12-14 at 11 56 36" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/692d11d2-058d-469b-a146-a4b92c33220f">

More in #43 

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
